sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/service/json/JsonConverterTest.java

```diff
@@ -40,7 +40,7 @@ public Map<String, String> getConfigOverrides() {
 
   private static final ObjectMapper mapper = new ObjectMapper();
 
-  @Inject private JsonConverter service;
```

Comment:
also here on line 41, inject the mapper, missed that in last PR..

Reply:
whoops mb